### ISSUE TITLE
Accessibility : Aztec Media Options TalkBack Improvements

### ISSUE THREAD

**Issue by jd-alexander** [wordpress-mobile/WordPress-Android#10894]
For this audit the governing principles are as follows ([inspired by a previous audit](https://github.com/wordpress-mobile/WordPress-Android/issues/7288))

- All the clickable elements should be marked as so.
- All nonactionable and decorative elements should be non-focusable.
- Relevant content should be grouped together.
- All important elements should be described through content description.

The following is a list of opportunities to improve the accessibility of the Editor's Media Options.

### TalkBack Issues

* Media Options Dialog
	* When you press the switch that opens the media options the main area that get's focused says "Out of Pager" I guess it's to signify that there's a view that's overlaying the main pager so the user knows to hover to discover the new current view. Possibly renaming the pager i.e. changing what it's announced as might make it clearer as to what the user is interacting with eg. renaming the pager's announcement to "Editor"

* Gallery
	* Disable "Person reading device with notifications" announcement on the permissions view within the media options dialog.
	
* Free Photo Picker
	* "List" is read by the announcer.
	* Descriptions provided for images here are really good. Good to replicate for other pickers. 

* WordPress Media Picker
	* Search - Double check to see what actually powers the media search since the filenames aren't being found during the process when using TalkBack.

**Comment by shiki** [comment]
Great audit! I also noticed some things in the Media Library.

<img src="https://user-images.githubusercontent.com/198826/70104845-93924500-15fc-11ea-81f6-5a03a5eba294.png" width="320">

- The item's check/unchecked status is not announced
- The filename being spoken may also not be very useful and can sometimes be very long. If the check/unchecked status is announced before the filename, then this is probably okay.
- The ✖️ is labeled as “Done” and the ✔️ is labeled as “Confirm”. These seem quite vague when they are shown together.

**Comment by jd-alexander** [comment]
Thanks for spotting these. 

> The filename being spoken may also not be very useful and can sometimes be very long. If the check/unchecked status is announced before the filename, then this is probably okay.

Yes, I noticed this behavior in some other pickers. With the Free Photo Picker, the provided descriptions are good but I guess that is because the source that's providing them have good filenames.
